evap/evaluation/models.py

```diff
@@ -1094,7 +1095,8 @@ class Question(models.Model):
     HEADING = 5
     QUESTION_TYPES = (
         (_("Text"), ((TEXT, _("Text question")),)),
-        (_("Unipolar Likert"), ((LIKERT, _("Agreement question")),)),
+        (_("Unipolar Likert"), ((POSITIVE_LIKERT, _("Agreement question")),)),
+        (_("Unipolar Likert"), ((NEGATIVE_LIKERT, _("Negative agreement question")),)),
```

Comment:
"Disagreement question"? @janno42 @He3lixxx 

Furthermore, "unipolar likert" now appears twice, which seems wrong

Reply:
Quote from the issue:
> The current 'LIKERT' scale should be renamed to "POSITIVE_LIKERT" ("Positive agreement") and the new scale will be "NEGATIVE_LIKERT" ("Negative agreement").

I agree with that (positively).